README.md

```diff
@@ -43,6 +43,53 @@ Config can have **various sources** to sync from. When a source is configured, t
 
 ## Migrations
 
+### Existing Zitadel deployments
+
+For Zitadel deployments that have not been synced to before, we
+provide an "ID installation" binary. This links users to their LDAP
+counterparts via the LDAP ID, so that famedly-sync can correctly
+identify them in subsequent sync runs.
```

Comment:
I would still include up here right at the beginning that this migration is intended to be used EXACTLY ONCE, as well as the criteria for determining whether this should be used.

Reply:
Also some information ab out what can go wrong when using it and how to respond to problems